affine/tests/test_transform.py

```diff
@@ -46,33 +49,33 @@ def seq_almost_equal(t1, t2, error=0.00001):
 
 class PyAffineTestCase(unittest.TestCase):
```

Comment:
This class no longer needs to exist, but I didn't want to make the diff harder to read.